### ISSUE TITLE
Inconsistent download path to helm3

### ISSUE THREAD

**Issue by shaneing** [alexellis/arkade#205]
## Expected Behaviour

`arkage get helm` download the helm v3 to `/home/ubuntu/.arkade/bin/helm3/helm`.

## Current Behaviour

`arkage get helm` download the helm v3 to `/home/ubuntu/.arkade/bin/helm`. But, it will download helm v3 to `/home/ubuntu/.arkade/bin/helm3/helm` when I install an app with `--helm3`. I suggest that keep it consistent.

## Steps to Reproduce (for bugs)

1. arkade install openfaas --helm3--gateways=2
2. arkage get helm

## Your Environment

* What Kubernetes distribution are you using?

```
kubectl version
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.6+k3s1", GitCommit:"6f56fa1d68a5a48b8b6fdefa8eb7ead2015a4b3a", GitTreeState:"clean", BuildDate:"2020-07-16T20:46:15Z", GoVersion:"go1.13.11", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.6+k3s1", GitCommit:"6f56fa1d68a5a48b8b6fdefa8eb7ead2015a4b3a", GitTreeState:"clean", BuildDate:"2020-07-16T20:46:15Z", GoVersion:"go1.13.11", Compiler:"gc", Platform:"linux/amd64"}
```

* Operating System and version (e.g. Linux, Windows, MacOS):

```
uname -a
Linux k3s 5.4.0-42-generic #46-Ubuntu SMP Fri Jul 10 00:24:02 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux


cat /etc/os-release
NAME="Ubuntu"
VERSION="20.04.1 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.1 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
```

* What arkade version is this?

```
arkade version
            _             _
  __ _ _ __| | ____ _  __| | ___
 / _` | '__| |/ / _` |/ _` |/ _ \
| (_| | |  |   < (_| | (_| |  __/
 \__,_|_|  |_|\_\__,_|\__,_|\___|

Get Kubernetes apps the easy way

Version: 0.6.10
Git Commit: 08ce44c5a23ff95576f7ccdc7e6ea956a6cbd85e
```

**Comment by amalkh5** [comment]
I think this issue is fixed by #208

**Comment by Waterdrips** [comment]
/close

This is indeed fixed by#208
